README.md

```diff
@@ -13,37 +12,36 @@ The Codacy extension for IntelliJ IDEA helps you review and manage the issues fo
 
 Before installing the extension, make sure you meet the following requirements:
 
-1. You have a [Codacy account](https://www.codacy.com/signup-codacy).
-2. The repository you’re working on has been [added to Codacy Cloud](https://docs.codacy.com/organizations/managing-repositories/#adding-a-repository).
-3. You have at least [Repository Read permissions](https://docs.codacy.com/organizations/roles-and-permissions-for-organizations/) for the repository you’re working on.
-4. IntelliJ IDEA, version 2023.3.3 or later.
+1.  You have a [Codacy account](https://www.codacy.com/signup-codacy).
+2.  The repository you’re working on has been [added to Codacy Cloud](https://docs.codacy.com/organizations/managing-repositories/#adding-a-repository).
+3.  You have at least [Repository Read permissions](https://docs.codacy.com/organizations/roles-and-permissions-for-organizations/) for the repository you’re working on.
+4.  You're using IntelliJ IDEA version 2023.3.3 or later.
```

Comment:
I'm not a fan of hardcoding version numbers unless strictly necessary.

If this is the same version mentioned in [gradle.properties](https://github.com/codacy/codacy-intellij-extension/blob/master/gradle.properties#L15) and the IDE check this when installing, I suggest removing this line.

Or are there other use cases I'm not aware of that justify this?

Reply:
We don't have it for VSCode, no reason to have it here. Feel free to remove it.